src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -7015,20 +7015,20 @@ private enum ParseTypeMode
             FirstElementOfPossibleTupleLiteral,
         }
 
-        private TypeSyntax ParseType(ParseTypeMode mode = ParseTypeMode.Normal)
+        private TypeSyntax ParseType(ParseTypeMode mode = ParseTypeMode.Normal, bool isTopLevelSwitchPattern = false)
```

Comment:
can you doc what 'isTopLevelSwitchPattern' means?  i'm not really understanding the 'top level' portion of things.